### PR TITLE
UCS/TEST: Add string_buffer data structure

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucs/datastruct/string_buffer.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_DYNAMIC_STRING_H_
```
Comment (hoopoepg) [inline src/ucs/datastruct/string_buffer.h]: macro is not file name based?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11924/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11923/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 21 workers (click for details)</summary>

**Note:** the logs will be deleted after 22-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/8136faabbab0012aca5738e79f2205c26f74ef75/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/3a565af1ed4f1dd97e687217681a79e87b2df94e/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/f7ea318a32883ee55e1e339c474c13eb3fa7f7b4/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/0e5a9f8c4c3cbf075c756c96305377eafe28d62d/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/ae6051f698c67fc69d36963f870bbbe45f967df7/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/f617ee2e788fc64ed8185371ea9305344e5e4e89/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/2f517325cee96b98dedc057f0afeb3b43347de59/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/6b68d973dfc6e744ee0ff50d2cd5c3225bed54e3/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/141eb410cf14d728bdd703e2738f3633dbb3bd82/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/218b292612ca2b2a395265218d81dd1fab0767b0/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/106b529d7a2effaaa1e89339500097fdbbd40516/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/3f639dfa1cd8e106bf8e460b7b4e24ecec13adf2/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/ed1687ccfbd0d7e7a2f72077a3a97bd8a53093e4/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/7010823c0b00a6cbef8ebb2918a4d10ae72928f6/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/9d57aa36ca55d3f735bbfc7de135d02fa2d0c1ad/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/58575cc838c67ac20ed38a36dbf216aee90fc0c7/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/adc30e259343921cae02ed2d66f9580962e79338/hpc-test-node-legacy_W3) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/43fcda036524f67270017484a9c172c335172e12/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/7f26ac74e8642915525647dfcbe9963db428919d/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/2c55a7a2aeb4ff38a60f20e86f8f8d66804c3d74/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/e74420fde397864cb6208adae3b549eb/raw/6119e0ef897d143ccfc19e208b4086a41a379a12/hpc-test-node-new_W3) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 23 workers (click for details)</summary>

**Note:** the logs will be deleted after 22-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/8a0756092dd7b9243c9cffc6532a74f9684315a3/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/e2af7cb036213de04928e05268dad6131b626db7/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/d9a60b0dff0679790c7b5fe187935d18b0e47815/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/d3903bd816c41a41f734e98858fc7ae7a2129e5d/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/0e3b12f8c18cb246f00928f9695354b64fb06fe3/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/6683666696d8a829de938eb333081c56d3dec1ef/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/c346616f8c3b272dfe4f5ebf825c9a23d133b405/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/013c7417b925ee3bb74a736b548627faa36c697d/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/415f107f27b71f15a295eaa4a33f7aa43707a1f1/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/dacf600c9d85eb2aeb76ca32cd4e4c421e700e57/hpc-test-althca_W0) | :question: UNKNOWN
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/90325d0da7933cb17d6c3d52390bfde427f4a315/hpc-test-althca_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/4ddc485fa8d1e0faa90b8fd15d8510ab30b00ce2/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/05718730fbd2a675ccca6a58b6af93b3ae9bbb90/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/034a38de5ef082ebafab38390c9b538b537ee237/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/9120b9f3e71df9ea4e62305ed5c48af58c3ceea7/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/5c224b8250df6b8812f06a2b125e22fe54cbb2b9/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/de92f53b2a1ea44d0b7e539faa1af9b6d105bd80/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/1318f64b91812641b51f2705efda7e75ff0bfb89/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/5c4ebfbbda261e546ed7b5a7e1ea157aeee0167e/hpc-test-node-legacy_W3) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/0e330c155801343c6aed72dae14bfb9bc0b6e5ba/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/1c91474e9595bf5858daf37f06d27b5896cb2ac8/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/684af04b8192798822253a28b17d60ed8f61e103/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/cfa02c67ee927b2623a7ff4ddd77e552/raw/0fb34f0d86b1ecafbdda2c81d3466f9bb458c7cd/hpc-test-node-new_W3) | :question: UNKNOWN

File: src/ucs/datastruct/string_buffer.c [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    va_start(ap, fmt);
+    max_print = strb->capacity - strb->length - 1;
+    ret       = vsnprintf(strb->buffer + strb->length, max_print, fmt, ap);
+    va_end(ap);
+
+    /* if failed, grow the buffer to at least the required size and at least
+     * double the previous size (to reduce the amortized cost of realloc) */
+    if (ret >= max_print) {
+        status = ucs_string_buffer_grow(strb, ucs_max(strb->capacity * 2,
+                                              strb->length + ret + 1));
```
Comment (hoopoepg) [inline src/ucs/datastruct/string_buffer.c]: alignment, again alignment :)

File: src/ucs/datastruct/string_buffer.h (hunk 1 above)
Comment (yosefe) [reply src/ucs/datastruct/string_buffer.h]: fixed, this eas previous name

File: src/ucs/datastruct/string_buffer.c (hunk 2 above)
Comment (yosefe) [reply src/ucs/datastruct/string_buffer.c]: fixed

**Comment by yosefe** [comment]
@hoopoepg static initializer could be done, but it was not needed for the end goal (verbose message from UCP about all available devices and TLs)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11925/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 15 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 22-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/2180e4d83a581750f0c7f95692b6d9a3a2f19834/_main) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/5089858b0f12de20337f7e892b2fe1b391438428/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/53e04b054219819b8ce40e94b8e45a9c88d79f6e/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/1f3434b0e1b30d485a4d846c85f6c9a0af134beb/hpc-test-althca_W0) | :x: FAILURE
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/ae64d0b79fd3e8d6350d4e55d0f4ae8a9874dffc/hpc-test-althca_W2) | :x: FAILURE
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/b5f806ba32c057fe20337499aa261fc63284e14f/hpc-test-althca_W3) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/598ba456a4b083eefaf8a1697f6c688056673625/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/7572379e770a0e3f7899ab5ed096281c809d805e/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/1d95ef457eafc56be48fa2702629678a6e1ae6e2/hpc-test-node-legacy_W3) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/607facc9db45ed53368da53220372d24d6299a3e/hpc-test-node-new_W0) | :x: FAILURE
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/b71c6b12905e53baf5cf7973658630f48a6fa6fe/hpc-test-node-new_W1) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/3519110943346c491ce4397634d03b697659b3c7/hpc-test-node-new_W2) | :x: FAILURE
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/15fafa493ae59546eb24498887133e02638da8da/hpc-test-node-new_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/eb344f598046af8326daaea862da026d7f05b4c7/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/4fdf3b6d1520a96663a44ebc5f9e698a4ad6766d/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/6bbcd9c194e4c1c67c273d5e9b267112d149606b/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/e1c62a3b5797084ca5f6a27374a2315d333b32b5/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/f00fedb9c8b188fc7c3cd227f69a50b1837cc5c6/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/5890c5a47f7f8db1fd3435e13637b2ce97a546ee/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/3075b551075e73f642a8aa8b04676d9c0be0ed0d/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/7fdb1cfb60921a02175b44044693c9573b1d5ff7/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/d6debdf9de91d230120399cfc1fe271ce3868601/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/366f4f5584843f3de1a75e40266c9cc47cae8c13/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/330f46e0898ca81585b97ce8e8b01fc06f898383/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/34f9a9535279fc39e32cb3f02d498e4c13d12ea0/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/df281bbe33996ee5eff1c957caf2131794712876/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/7b509e29c8f3ea8e10acb47cded14674d92b6c1f/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/75c7b306ff82c9196847d1e0c678dc3af1be3648/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4a1397aa6b6bc9a3d15910bea9ee551b/raw/ab458f200008a93ca11beb7c68c2d4fb2b90c01f/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11926/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 22-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/f004b300080d20d82873d3f3ec144d160b4779a8/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/b99f9d9cec87ef6a9ec3fbb0e5ab2219c2db477c/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/663a1ee5c125b8cd5b9ee5fbb69285ca424b1ecd/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/35845d147c7b13f2381162cb0ef9eda7bc38e6ac/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/01ead45a79d28386bb5a2f1a9a5f989728183394/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/2b02acc1ea46cee50b9f290a819056ae5fdde642/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/367ccb05ec2e0a049488e3200d35d6b6019fa7a3/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/8f4c8cd1365a3214b79f2f3e559e8c8d4a9ae4b1/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/11c32831f25728cdf7880337275aaa309a3f6010/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/0603a4d556f61dc55f4ac3b2f2a3f390ada965e6/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/a3cbc95b9820f7024afeac4fb26483d49961e45f/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/c9d9688144706ea8b7c828393023c4a4b1a23851/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/0d7bf5e7a133290648be9c9d0268a1ad4fe3ec8f/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/8cd7673038bc93b5332fb914e471da8cd2696c10/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/18fad0c4b847dc2334ec88738e2252cb7ba6b856/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/4f918cb416366fcc5c6a003630ba90ffeb7735ea/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/ecb41b25c87c4000c35a1c6baf718427dfd75eb3/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/ced218d9ca5b6d531e41d971731521da44f3ab0a/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/10c28c571514689c31a7abce3c4605232f512323/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/47d4fdcfe54c938d785fe58da49b587eed556a0f/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/ab62f63fe45078557dfa88cedb8903ec21f6f0d5/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/dfac624457c1d8d46b4937628e02af52718fbf6a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/5806286b70c0480de6518a141511578b907c3209/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/12f210ca35e970addc775c76e0c10fcf3c2a95d2/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/9ae117b528a4f8a7fe1c42bc5377067089a1c944/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/70e60f3edac00549ebb18d46e853ed09cea82c6d/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/76cdab74a3e468cf71d006ab70df8e7b0d46a9c4/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/64b7e3e08d4ef3ac847c9dca2d9935cd45556ac3/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/03f7d09313ffb17d22d2625a5eddccba/raw/c7092ac9507f447b1069bd87e49748540fe0160f/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
bot:pipe:retest

**Pull request merged**